ads/google/imaVideo.js

```diff
@@ -1123,6 +1125,16 @@ function onMessage(global, event) {
       }
       postMessage({event: VideoEvents.UNMUTED});
       break;
+    case 'hideControls':
+      if (!adsActive) {
```

Comment:
If it does both `preventDefault` and `stopPropagation` (per `onProgressClick` above), won't it effectively prevent triggering ads anyway?

Reply:
The idea here is that requests to hide/show controls are not honoured during ad-play. `preventDefault` and `stopProgpagation` simply stop horizontally scrollable ancestors ( like `amp-carousel` and Viewers) to move when `seekBar` is horizontally moved.

Reply:
Oh gotcha.